src/hashmap.hpp

```diff
@@ -0,0 +1,55 @@
+#include <fstream>
```

Comment:
Can you prefix them with your project's name?

The issue with generic include guard names such as `HASHMAP_H` is the probability for name clashes is high. If a header you include (or a header your included header include) defines this symbol your include guard is broken. That's the reason we use e.g. `OSRM_SOME_HEADER_NAME_H` in osrm.